src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -56,4 +59,21 @@ public SendLetterController(
 
         return ok().body(messageId);
     }
+
+    @GetMapping(path = "/{message-id}")
+    @ApiOperation(value = "Get letter status")
+    @ApiResponses({
+        @ApiResponse(code = 200, response = LetterStatus.class, message = "Success"),
+        @ApiResponse(code = 401, message = "Invalid service authorisation header"),
+        @ApiResponse(code = 404, message = "Letter not found")
+    })
+    public ResponseEntity<LetterStatus> getLetter(
```

Comment:
Not being too choosy about the name but I think we can name the method as `getLetterStatus`

Reply:
yes we can. i had this name as a starting point but not updated after all code composed into this state